config/config.go

```diff
@@ -1463,3 +1467,26 @@ func isValidCookieSize(maxCookieSize int) error {
 	}
 	return nil
 }
+
+// IsAccountEventEnabled checks whether events are enabled for an account. In some incoming requests,
+// the `account.events_enabled` field, which is deprecated, is still used. We want to use its value whenever
+// the `account.events.enabled` field is not found.
+func IsAccountEventEnabled(deprecated_events_enabled *bool, events_enabled *bool) bool {
```

Comment:
As mentioned in the `config/config_test.go` comment, we should expect `false` in the 4th unit test of `TestIsAccountEventEnabled`. For this outcome to be met, this function needs refactoring. 

In a nutshell, want `IsAccountEventEnabled` to return the `events_enabled` value if set. If not, we should check if our second option, `deprecated_events_enabled` is set and return that. If none of the flags was set, we should return `false`.